src/Model/Parser.cs

```diff
@@ -13,7 +13,7 @@ internal static class Parser
 
         public static ITry<decimal, IEnumerable<string>> ParseDecimal(string value)
         {
-            var isSuccess = Decimal.TryParse(value.Replace('.', ',').Replace('−', '-').Trim(), NumberStyles.Number, FrenchCulture, out var result);
+            var isSuccess = decimal.TryParse(Regex.Replace(value, @"\s+", "").Replace('.', ',').Replace('−', '-'), NumberStyles.Number, FrenchCulture, out var result);
```

Comment:
Why do we need to remove string characters now? I'd expect all of those columns should only contain valid decimals?

Reply:
🤦‍♂️ Just removes spaces, my bad. All good.